### PR TITLE
Added ability to configure approval_prompt type

### PR DESCRIPTION
**Pull request by naveed-ahmad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,22 +12,26 @@ gem "omniauth-strava"
...
 Run bundle install afterwards.
 
-
 To ``config/initializers/omniauth.rb`` add:
 
-
 ```ruby
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :strava, Rails.application.secrets.strava_client_id, Rails.application.secrets.strava_api_key, scope: 'public'
+  provider :strava, 'strava_client_id', 'strava_api_key', scope: 'read', approval_prompt: 'auto'
```
Comment (thogg4) [inline README.md]: Is there a reason to change these calls to secrets to strings?

Comment (naveed-ahmad) [reply README.md]: Rails has secrets, encrypted credentials, ENV variable and what not for configuring. Shouldn't we let user decide which approach they want to go with?

Comment (thogg4) [reply README.md]: Yep, I agree they should be able to choose, that is why we don't enforce getting those values from a specific place.
I will be happy to merge this when its clear that they need to set those values themselves. I don't think just having strings does that.

Comment (naveed-ahmad) [reply README.md]: Will **STRAVA_CLIENT_ID_HERE** and **STRAVA_CLIENT_SECRET** do? 

I mean these static keys are obvious, devs know these are placeholders. All omniauth provider do the kind of same thing. 

https://github.com/rikkipitt/omniauth-affinitylive
https://github.com/arthurnn/omniauth-500px
https://github.com/anmolarora/omniauth-adroll

Comment (thogg4) [reply README.md]: Yeah that works great 👍 I don't mean to be too picky about this, I just want it to be easy and obvious to use. Thanks for your work!

**Pull request closed without merging**